hmrc-mongo-play-26/src/main/scala/uk/gov/hmrc/mongo/cache/collection/model.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mongo.cache.collection
+
+import java.time.LocalDateTime
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, __}
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+final case class CacheItem[A](
+  id: String,
+  data: A,
+  createdAt: LocalDateTime,
+  modifiedAt: LocalDateTime
+)
+
+object CacheItem {
```

Comment:
Move the formatting into `object PlayMongoCacheCollection` and make private? I don't think clients should be able to couple to the internal formatting.